salt/cni/policy/default-kube-dns-policy.yaml.jinja

```diff
@@ -0,0 +1,17 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: kube-dns-policy
+  namespace: kube-system
+spec:
+  podSelector:
+    matchLabels:
+      "k8s-app": "kube-dns"
+  ingress:
```

Comment:
Don't we need some egress rules here too?

Reply:
Ingress should be enough given that we trust our application/container.